### PR TITLE
Upgrade spree dependency

### PR DESCRIPTION
**Pull request by mbj**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -4,7 +4,7 @@ SPREE_OPTS = [
   '= 2.4.11.beta',
   {
     git: 'https://github.com/MountainRoseHerbs/spree.git',
-    ref: 'df5e12d'
+    ref: '1e7fba61e'
```
Comment (dkubb) [inline Gemfile]: FYI, when run from my local spree repo:

```bash
$ git rev-parse --short HEAD
1e7fba6
```

Comment (mbj) [inline Gemfile]: Thats interesting. Did you GC your repo recently? Maybe git internally does use the longer format only when certain size of `.pack` files exist?

Comment (dkubb) [inline Gemfile]: I did use `git gc --aggressive` this morning.

Comment (mbj) [inline Gemfile]: We discovered in channel this is a version difference between our environments git.

Comment (dkubb) [inline Gemfile]: Maybe we should use github's UI as a source of truth:

screen shot 2016-12-21 at 6 01 58 pm

Comment (mbj) [inline Gemfile]: I'd rather use something "local". And frankly: We should IMO use the non abbreviated hashes anyway. As only these guarantee future git states are still uniquely referencing a specific commit when running bundler on the old state again.

**Pull request merged**